source/backends/ubuntu/ubupkg.d

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ * Author: Iain Lane <iain.lane@canonical.com>
+ *
+ * Licensed under the GNU Lesser General Public License Version 3
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the license, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this software.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+module backends.ubuntu.ubupkg;
+
+import appstream.Component;
+
+import backends.debian.debpkg;
+import bindings.gettext;
+import backends.interfaces;
+
+import glib.Internationalization;
+import glib.KeyFile;
+
+import std.container : Array;
+
+import logging;
+
+immutable DESKTOP_GROUP = "Desktop Entry";
```

Comment:
On Tue, Aug 30, 2016 at 08:31:34AM -0700, Matthias Klumpp wrote:
> > +module backends.ubuntu.ubupkg;
> > +
> > +import appstream.Component;
> > +
> > +import backends.debian.debpkg;
> > +import bindings.gettext;
> > +import backends.interfaces;
> > +
> > +import glib.Internationalization;
> > +import glib.KeyFile;
> > +
> > +import std.container : Array;
> > +
> > +import logging;
> > +
> > +immutable DESKTOP_GROUP = "Desktop Entry";
> 
> No sure what you mean...
> `public immutable DESKTOP_GROUP = "Desktop Entry";` and then `import utils : DESKTOP_GROUP` wherever you need it.

(why don't the emails quote my text too?)

You asked about using the value in GLib, but we can't because it's not
exposed in the bindings. So I moved the definition to utils.

-- 
Iain Lane                                  [ iain@orangesquash.org.uk ]
Debian Developer                                   [ laney@debian.org ]
Ubuntu Developer                                   [ laney@ubuntu.com ]